velox/exec/SetAccumulator.h

```diff
@@ -205,16 +230,19 @@ struct ComplexTypeSetAccumulator {
   }
 
   vector_size_t extractValues(BaseVector& values, vector_size_t offset) {
-    vector_size_t index = offset;
+    vector_size_t numValues = 0;
     for (const auto& position : base.uniqueValues) {
-      AddressableNonNullValueList::read(position, values, index++);
+      AddressableNonNullValueList::read(
+          position.first, values, offset + position.second);
+      numValues++;
```

Comment:
no need to increment this on each loop iteration; just

auto numValues = base.uniqueValues.size();